core/templates/dev/head/services/EditabilityServiceSpec.js

```diff
@@ -0,0 +1,52 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Test for EditabilityService.
+ */
+
+describe('EditabilityService', function() {
+  var EditabilityService;
+
+  beforeEach(module('oppia'));
+
+  beforeEach(inject(function($injector) {
+    EditabilityService = $injector.get('EditabilityService');
+  }));
+
+  it('allows to edit exploration if tutorial ends and exploration is editable',
+    function(){
+      EditabilityService.onEndTutorial();
+      EditabilityService.markEditable();
+      expect(EditabilityService.isEditable()).toBe(true);
+    });
+
+  it('does not allows to edit an exploration, if it is in tutorial mode',
+    function(){
+      EditabilityService.onEndTutorial();
```

Comment:
Looks similar to the above case. (If it's in the tutorial mode I think it should not be `onEndTutorial`)